### PR TITLE
Improve getting started

### PR DESCRIPTION
**Pull request by danielchg**

### CODE REVIEW

File: content/en/docs/KRE/getting_started/index.md [hunk 1]
```diff
@@ -4,24 +4,20 @@ description: A quick guide to start working with KRE.
...
-In order to work with KRE you will need these requisites. This is not completely mandatory but if you read all docs before starting, it would help to understand better what you are doing on each step.
-
-- [Basic helm knowledge]({{< relref "docs/KRE/installation/helm" >}})
-- [KRE basic concepts]({{< relref "docs/KRE/concepts" >}})
-
-
 ## Installation
 
-To start working with KRE you first need to get it installed in some Kubernetes cluster. You can choose different installation targets like [local Minikube]({{< relref "docs/KRE/installation/minikube" >}}), [EKS]({{< relref "docs/KRE/installation/EKS" >}}) or [GKE]({{< relref "docs/KRE/installation/GKE" >}}). 
+The best way to start working with KRE, to test it and understand how it works, is deploy it on your local machine and follow a basic instructions to create an example project.
```
Comment (Sergiodfdez) [inline content/en/docs/KRE/getting_started/index.md:9]: change:
-  is deploy for is **to** deploy and a basic instructions for basic instructions '

File: content/en/docs/KRE/installation/local/index.md [hunk 2]
```diff
@@ -1,14 +1,26 @@
...
+weight: 10
 ---
 
-# Minikube deployment
+# Requirements
 
-Minikube implements a local Kubernetes cluster on your OS. It runs the latest stable release of Kubernetes, with support for all standard Kubernetes features.
+The recomended way to test KRE is deploy it on top of Minikube with VirtualBox driver. Deploying this way you can check a full featured installation of KRE.
+
+Install the below required software following the steps in their respective sites, depend of your OS you will need one method or other described on each one.
```
Comment (guumaster) [inline content/en/docs/KRE/installation/local/index.md:13]:
```suggestion
Install all these required software by following the guides linked bellow.
```

File: content/en/docs/KRE/installation/_index.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ---
 title: "Installation"
 linkTitle: "Installation"
-weight: 20
+description: How to deploy KRE don ifferent flavors of Kubernetes.
```
Comment (guumaster) [inline content/en/docs/KRE/installation/_index.md:4]:
```suggestion
description: How to deploy KRE on different flavors of Kubernetes.
```

File: content/en/docs/KRE/getting_started/index.md [hunk 4]
```diff
@@ -33,9 +29,9 @@ You need to follow these steps in order to create a version and get it ready to
 Once you are logged in as an administrator, you can create a new Runtime by simply following the [create Runtime guide]({{< relref "docs/KRE/tasks/create_runtime" >}}).
 
 
-### 2. Create a KRT file
+### 2. Download an sample KRT file
```
Comment (guumaster) [inline content/en/docs/KRE/getting_started/index.md:32]:
```suggestion
### 2. Download a sample KRT file
```

File: content/en/docs/KRE/getting_started/index.md [hunk 5]
```diff
@@ -33,9 +29,9 @@ You need to follow these steps in order to create a version and get it ready to
...
 
-Inside a Runtime you can upload one or more versions that can later be ran and publish. You can use a pre-generated [sample greeter krt](/website/krts/greeter-v1.krt), or learn [how to create your own KRT file]({{< relref "docs/KRT/tasks" >}}).
+Inside a Runtime you can upload one or more versions that can later be ran and publish. You can use a pre-generated [sample greeter krt](/website/krts/greeter-v1.krt). You can find a more detailed explaination of how to create your own KRT file [here]({{< relref "docs/KRT/tasks" >}}).
```
Comment (guumaster) [inline content/en/docs/KRE/getting_started/index.md:34]:
```suggestion
Inside a Runtime you can upload one or more versions that can later be ran and publish. You can use a pre-generated [sample greeter krt](/website/krts/greeter-v1.krt). You can find a more detailed explaination about how to create your own KRT file [here]({{< relref "docs/KRT/tasks" >}}).
```

File: content/en/docs/KRE/getting_started/index.md (hunk 1 above)
Comment (guumaster) [inline content/en/docs/KRE/getting_started/index.md:9]:
```suggestion
The best way to start working with KRE, to test it and understand how it works, is deploying it on your local machine and follow some basic instructions to create an example project.
```

**Review by Sergiodfdez — APPROVED** [review]

File: content/en/docs/KRE/installation/cloud/_index.md [hunk 6]
```diff
@@ -0,0 +1,6 @@
+---
+title: "Cloud"
+linkTitle: "Cloud"
+weight: 30
+---
+Following are described the best practice to deploy KRE on most common Kubernetes solution by Cloud providers.
```
[the remark this answers is not in the document]
Comment (guumaster) [reply content/en/docs/KRE/installation/cloud/_index.md:6]: Best Practices is the correct form.

File: content/en/docs/KRE/installation/local/index.md [hunk 7]
```diff
@@ -1,14 +1,26 @@
...
-weight: 20
+  How to install KRE on your local machine for testing or development purpose.
+weight: 10
 ---
 
-# Minikube deployment
+# Requirements
 
-Minikube implements a local Kubernetes cluster on your OS. It runs the latest stable release of Kubernetes, with support for all standard Kubernetes features.
+The recomended way to test KRE is deploy it on top of Minikube with VirtualBox driver. Deploying this way you can check a full featured installation of KRE.
```
[the remark this answers is not in the document]
Comment (guumaster) [reply content/en/docs/KRE/installation/local/index.md:11]: No, "Deploying this way" is the correct form.

**Pull request merged**